README.md

```diff
@@ -23,22 +23,14 @@ It is designed to reduce the maintenance surface, with a flexible approach to pr
 
 cOS derivatives are built from containers, and completely hosted on image registries. The build process results in a single container image used to deliver regular upgrades in OTA approach. Each derivative built with `cos-toolkit` inherits by default the [following featuresets](/docs/derivatives_featureset.md).
 
-cOS supports different release channels, all the final and cache images used are tagged and pushed regularly [to DockerHub](https://hub.docker.com/r/raccos/releases-amd64/) and can be pulled for inspection from the registry as well.
+cOS supports different release channels, all the final and cache images used are tagged and pushed regularly [to Github Container Registry](https://quay.io/repository/costoolkit/releases-opensuse) and can be pulled for inspection from the registry as well.
```

Comment:
```suggestion
cOS supports different release channels, all the final and cache images used are tagged and pushed regularly [to Quay Container Registry](https://quay.io/repository/costoolkit/releases-opensuse) and can be pulled for inspection from the registry as well.
```

Reply:
gah! :facepalm:  I thought I catched up all the occurences.. thanks!